### PR TITLE
Develop/abandoned cart

### PR DESCRIPTION
**Pull request by sinukaarel**
# Changes made to modify existing module

## Abandoned cart

- New feature abandoned cart reminder - send automatic reminder email with smaily template
- Table **smaily_cart** to hold sent abandoned cart data

## Module base

- All autoresponders are shown in autoresponders list
- Improved displaying autoresponders/form when allready validated credentials
- Changed cron display messages
- Version bump 1.1.0
- Translations for new features

### CODE REVIEW

File: views/js/smaily_module.js [hunk 1]
```diff
@@ -81,4 +81,66 @@ $(document).ready(function() {
       }
     });
   });
+  // If autoresponders allready validated call smaily api to populate autoresponders list
+  (() => {
```
Comment (kaittodesk) [inline views/js/smaily_module.js]: ES6 syntax does not work in older browsers. Is Prestashop not supported by older browsers and usage of ES6 is OK?

Comment (sinukaarel) [reply views/js/smaily_module.js]: Changed function to ES5 syntax.
d0b3e76d32877f3fe09afb0e0a1e5a83639205b1

Comment (kaittodesk) [reply views/js/smaily_module.js]: Isn't `let` also part of ES6 syntax?

Comment (sinukaarel) [reply views/js/smaily_module.js]: Went trough previous JS file and switched all to `var ` @kaittodesk

**Pull request merged**